vendor/immediate-observer.js

```diff
@@ -0,0 +1,32 @@
+(function() {
+  var _Ember;
+  if (typeof Ember !== 'undefined') {
+    _Ember = Ember;
+  } else {
+    _Ember = require('ember').default;
+  }
+
+  if (EmberENV && EmberENV._ENABLE_IMMEDIATE_OBSERVER_SUPPORT !== true) {
+    return false;
+  }
+
+  _Ember.immediateObserver = function() {
+    _Ember.deprecate('Usage of `Ember.immediateObserver` is deprecated, use `observer` instead.', false, { id: 'ember-metal.immediate-observer', until: '3.0.0' });
+
+    for (let i = 0; i < arguments.length; i++) {
+      let arg = arguments[i];
+      _Ember.assert(
+        'Immediate observers must observe internal properties only, not properties on other objects.',
+        typeof arg !== 'string' || arg.indexOf('.') === -1
+      );
+    }
+
+    return _Ember.observer.apply(this, arguments);
+  }
+
+  const EXTEND_PROTOTYPES = JSON.parse(decodeURIComponent(document.getElementsByName('dummy/config/environment')[0].content)).EmberENV.EXTEND_PROTOTYPES;
```

Comment:
@rwjblue I thought that global.EmberEnv contains the merged configs. I see the correctly merged content inside of the meta tag but not of the global. Do you know of a way to grab this from within vendor?

Reply:
Disregard this. Issue was with https://github.com/ember-cli/ember-disable-prototype-extensions